.travis.yml

```diff
@@ -4,7 +4,13 @@ cache: bundler
 rvm:
   - 1.9.3
   - 2.0.0
-  - 2.1.0
-  - 2.2.0
-  - 2.3.0
+  - 2.1
+  - 2.2
+  - 2.3
+  - 2.4
+  - 2.5
+  - 2.6
```

Comment:
Are we being a bit keen here? 2.6 is still in preview

Reply:
Meh I figured we may as well test against it anyway, to get advance warning of any issues. It's the point of being "in preview" I guess..?

Reply:
And I guess it's passing in Travis?

Reply:
So far...